vcs/internal/path.go

```diff
@@ -0,0 +1,13 @@
+package internal
+
+import "strings"
+
+// Rel strips the leading "/" prefix from the path string, effectively turning
```

Comment:
Can you please mention that this path string is a "forward-slash separated" path string, not a OS filesystem path string. Meaning it's always going to be "/" for root element, and never "\" or "C:" or something else.